QUARKUS-5613.md

```diff
@@ -0,0 +1,85 @@
+## Links to the resources
+- JIRA issue: https://issues.redhat.com/browse/QUARKUS-5613
+- GH PRs related to the Jira issue: https://github.com/quarkusio/quarkus/pull/43283 https://github.com/quarkusio/quarkus/issues/42117 
+- Guides modified and related:
+  https://quarkus.io/guides/security-openid-connect-client-reference
+  https://quarkus.io/guides/security-oidc-code-flow-authentication
+  https://quarkus.io/guides/security-openid-connect-client-registration
+- TP/TD PRs:
```

Comment:
```suggestion

```
not necessary, if anyone needs to track it down, all you need to do is to link TD to the Quarkus JIRA ticket.

Reply:
also, linking TP PR :-)